client/src/components/ReviewsFromUS.jsx

```diff
@@ -1,15 +1,254 @@
+  font-size: 14px;
+`;
+
+    const VerifiedPurchase = styled.span`
+`;
+
+    const ReviewPanelCollapsible = styled.div`
+    overflow: hidden;
+    position: relative;
+    font-family: "Amazon Ember",Arial,sans-serif;
+    font-size: 14px;
+    /* min-height: 50px; */
+`;
+    const ReviewPanelReadMore = styled.div`
+    color: #007185;
+    cursor: pointer;
+    font-family: "Amazon Ember",Arial,sans-serif;
+    font-size: 14px;
+    margin-bottom: 10px!important;
+   `;
+
+    const HelpfulWrapper = styled.div`
+    display: table;
+    content: "";
+    margin-bottom: 15px!important;
+    font-family: "Amazon Ember",Arial,sans-serif;
+    /* line-height: 0;
+    font-size: 0; */
+    font-size: 14px;
+    line-height: 20px;
+    box-sizing: border-box;
+
+    `;
+
+    const HelpfulCount = styled.div`
+    color: #565959!important;;
+    cursor: pointer;
+    font-family: "Amazon Ember",Arial,sans-serif;
+    font-size: 14px;
+    line-height: 20px!important;
+    box-sizing: border-box;
+   `;
+
+    const HelpfulButton = styled.button`
+   margin-right: 5px;
+   vertical-align: middle;
+   float: left!important;
+   box-sizing: border-box;
+   display: block;
+   font-family: "Amazon Ember",Arial,sans-serif;
+   color: #0F1111;
+   font-size: 14px;
+  line-height: 20px;
+ `;
+
+    const ReportAbuse = styled.span`
+    color: #565959!important;
+    font-size: 14px!important;
+    line-height: 20px!important;
+    font-weight: 400!important;
+    font-style: normal!important;
+    text-transform: none!important;
+    text-decoration: none!important;
+    font-family: "Amazon Ember",Arial,sans-serif;
+    cursor: pointer;
+    box-sizing: border-box;
+    `;
+
+
+    return (
+      <div >
+        <div>
+          <h3>Top reviews from the United States</h3>
+          {reviews.map(review => {
+            return (
+              <div>
+                <Row>
+                  <ProfileAvatarWrapper>
+                    <ProfileAvatar><img src={review ? 'https://fec-customers-bucket.s3-us-west-1.amazonaws.com/profile1.jpg' : null} /></ProfileAvatar>
+                  </ProfileAvatarWrapper>
+
+                  {/* <div><img className="profile-img" src="https://homepages.cae.wisc.edu/~ece533/images/pool.png" alt="profilePic" /></div> */}
+                  <ProfileContent>
+                    <ProfileName >{review ? review.customerName : null}</ProfileName>
+                  </ProfileContent>
+
+                </Row>
+
+
+                <div> <StarRating
+                  name="rate1"
+                  starCount={5}
+                  value={review ? review.rating : 0}
+                  starColor="#FFA500"
+                />
+                  <LetterSpace />
+                  <TitleSpan> {review ? review.title : null} </TitleSpan>
+                </div>
+
+                <ReviewDate>Reviewed in the {review ? review.customerCountry : null} on {review ? review.reviewDate : null}</ReviewDate>
```

Comment:
Are these going to be displayed on the page when there's no customer county or review date? It seems like they'll still show with missing data instead of maybe not being showed at all. Also, It seems like for a lot of these you're checking for the review object, maybe you could have a ternary if else in the outer scope and render some components only if you have the review object.